internal/output/output.go

```diff
@@ -140,58 +153,173 @@ func Combine(outs ...Root) Root {
 
 			totalMonthlyCost = decimalPtr(totalMonthlyCost.Add(*o.TotalMonthlyCost))
 		}
+
+		summaries = append(summaries, o.ResourceSummary)
 	}
 
-	sortResources(combined)
+	combined.sortResources()
 
 	combined.TotalHourlyCost = totalHourlyCost
 	combined.TotalMonthlyCost = totalMonthlyCost
 	combined.TimeGenerated = time.Now()
+	combined.ResourceSummary = combinedResourceSummaries(summaries)
 
 	return combined
 }
 
-func sortResources(out Root) {
-	sort.Slice(out.Resources, func(i, j int) bool {
-		return out.Resources[i].Name < out.Resources[j].Name
+func (r *Root) sortResources() {
+	sort.Slice(r.Resources, func(i, j int) bool {
+		return r.Resources[i].Name < r.Resources[j].Name
 	})
 }
 
-func skippedResourcesMessage(resources []*schema.Resource, showDetails bool) string {
-	summary := schema.GenerateResourceSummary(resources)
-	if summary.TotalUnsupported == 0 {
+func (r *Root) unsupportedResourcesMessage(showSkipped bool) string {
+	if r.ResourceSummary.UnsupportedCounts == nil || len(*r.ResourceSummary.UnsupportedCounts) == 0 {
 		return ""
 	}
 
-	supportedTypeCount := 0
-	for rType := range summary.UnsupportedCounts {
-		if terraform.HasSupportedProvider(rType) {
-			supportedTypeCount++
-		}
+	unsupportedTypeCount := len(*r.ResourceSummary.UnsupportedCounts)
+
+	pluralizedTypesMsg := "resource type"
```

Comment:
Maybe we should just remove this logic and always use `resource types` as in the message we use `them` vs `it` (we use to use `it` for singular).

Reply:
I've fixed it to show `them` vs `it` as well